src/components/actions/getApplicationData.ts

```diff
@@ -40,11 +41,19 @@ export const getApplicationData = async (input: {
   const reviewId =
     input?.reviewId ?? (trigger_payload?.table === 'review' ? trigger_payload?.record_id : null)
 
+  const reviewAssignmentId =
```

Comment:
Can you update the `ActionApplicationData` type in types.ts -- I don't' think it quite corresponds to this structure now.

Reply:
Nothing changed in the returned values. Is just allowing the action to run when received from event on `reviewAssignment` table, that's all.